### PR TITLE
refactor: use imported dark theme instead of hard-coded styles

### PR DESCRIPTION
**Pull request by amandaejohnston**
Replaces the hard-coded dark theme variables with imports. The `class` sheet was used to maintain existing functionality.

The Ionic dev build includes all the AA contrast work, and is needed here so the build passes (since otherwise the CSS imports don't resolve). It should be okay to merge into the `feature-8.0` branch since we'll need to replace it with v8.0 manually anyway.

To test locally, use `npm run dev`. Note that running `build` will trigger TS errors; these are unrelated to the PR and will be handled separately.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default defineComponent({
 
     const handleDarkModeChanged = (newDarkValue: boolean) => {
       dark.value = newDarkValue;
-      document.body.classList.toggle('dark', newDarkValue);
+      document.documentElement.classList.toggle('ion-theme-dark', newDarkValue);
```
Comment (amandaejohnston) [inline src/App.vue:39]: The `dark.class` stylesheet expects the class to be set on the root, not the body.

File: package.json [hunk 2]
```diff
@@ -18,8 +18,8 @@
     "@capacitor/keyboard": "5.0.6",
     "@capacitor/status-bar": "5.0.6",
     "@ionic/storage": "^4.0.0",
-    "@ionic/vue": "^7.0.0",
-    "@ionic/vue-router": "^7.0.0",
+    "@ionic/vue": "^7.6.2-dev.11704729332.13b412c8",
+    "@ionic/vue-router": "^7.6.2-dev.11704729332.13b412c8",
```
Comment (thetaPC) [inline package.json:22]: Is there a Jira ticket that tracks the removal of the dev builds?

File: src/theme/variables.css [hunk 3]
```diff
@@ -259,240 +259,3 @@
 .md {
 
 }
```
Comment (thetaPC) [inline src/theme/variables.css:261]:
```suggestion
}

```
Missing new line

**Review by thetaPC — CHANGES_REQUESTED** [review]
I noticed that the fab icon color has changed. `main` has it white while this branch has it black. I can't remember if this was intentional when the colors were selected?

| main | FW-5625 |
|--------|--------|
| Screenshot 2024-01-10 at 1 14 32 PM |   Screenshot 2024-01-10 at 1 14 24 PM |

**Comment by amandaejohnston** [comment]
> I noticed that the fab icon color has changed. `main` has it white while this branch has it black. I can't remember if this was intentional when the colors were selected?

Good catch 👀 Looking closer, the Angular and React conference apps have custom fab styling that changes it to a white background and primary color, but that styling is missing here. I've gone ahead and copied it in.

File: package.json (hunk 2 above)
Comment (amandaejohnston) [reply package.json:22]: Yup: https://ionic-cloud.atlassian.net/browse/FW-5873

File: src/theme/variables.css (hunk 3 above)
Comment (amandaejohnston) [reply src/theme/variables.css:261]: The file does have an ending newline now; the diff here just makes it hard to see.

File: src/views/SessionList.vue [hunk 4]
```diff
@@ -349,4 +349,18 @@ onMounted(() => {
 </script>
 
 <style scoped>
-</style>
\ No newline at end of file
+  ion-fab-button {
+    --background: var(--ion-color-step-150, #ffffff);
+    --background-hover: var(--ion-color-step-200, #f2f2f2);
+    --background-focused: var(--ion-color-step-250, #d9d9d9);
+    --color: var(--ion-color-primary, #3880ff);
```
Comment (thetaPC) [inline src/views/SessionList.vue:356]: Just verifying that this is the desired look?

| main | FW-5625 |
|--------|--------|
| Screenshot 2024-01-10 at 1 14 32 PM | Screenshot 2024-01-11 at 8 41 33 AM |

Comment (amandaejohnston) [reply src/views/SessionList.vue:356]: Yup, that's correct. If you look at the Angular and React conference apps, it looks the same as those.

**Review by thetaPC — APPROVED** [review]
LGTM

**Pull request merged**